rocky/rocky/templates/tasks/partials/task_filter.html

```diff
@@ -1,20 +1,46 @@
 {% load i18n %}
+<div class="filter">
+  <div>
+      <p>{% translate 'Filter' %}</p>
+      <button aria-expanded="false"
+              data-hide-filters-label="{% translate 'Hide filters' %}">
+        {% translate 'Show filters' %}
+      </button>
+  </div>
 
-{% if task_filter %}
-  <form method="get">
-    {% csrf_token %}
-    <label for="selected-tasks">{% translate 'Number of tasks showed' %}</label>
-    <select id="selected-tasks" name="selected-tasks">
-      {% for task in task_filter %}
-        <option {% if task == show_tasks %}selected{% endif %} value={{ task }}>
-          {% if task == "*" %}
-            {% translate 'Show all' %}
-          {% else %}
-            {{ task }}
+  <form action="" method="get" novalidate method="get">
+    <fieldset>
+        <label for="scan_history_from">{% translate 'From' %}</label>
+        <input type="date" name="scan_history_from" id="scan_history-from" value="{{ request.GET.scan_history_from }}">
+    </fieldset>
+
+    <fieldset>
+        <label for="scan_history_to">{% translate 'To' %}</label>
+        <input type="date" name="scan_history_to" id="scan_history-to" value="{{ request.GET.scan_history_to }}">
+    </fieldset>
+    <input type="submit"
+            value="{% if submit_text %}{{ submit_text }}{% else %}{% translate 'Set filters' %}{% endif %}"/>
+
+    <fieldset>
+      <label for="scan_history_status">{% translate 'Status' %}</label>
+      <select name="scan_history_status" id="scan_history_status">
+          <option value="" disabled selected>{% translate 'Select status' %}</option>
+          <option value="completed" {% if request.GET.scan_history_status == 'completed' %}selected{% endif %}>{% translate 'Success' %}</option>
```

Comment:
Is queued / dispatched also an option in the backend api? If so, we should add them too